ros2topic/ros2topic/api/__init__.py

```diff
@@ -57,8 +57,8 @@ def __call__(self, prefix, parsed_args, **kwargs):
 def import_message_type(topic_name, message_type):
     # TODO(dirk-thomas) this logic should come from a rosidl related package
     try:
-        package_name, message_name = message_type.split('/', 2)
-        if not package_name or not message_name:
+        package_name, *message_name = message_type.split('/')
+        if not package_name or not all(message_name):
```

Comment:
This wouldn't raise anymore if the input string doesn't contain any slashes?

Reply:
Ah, good catch. I thought `all` would handle that but doesn't. I can update this to make sure we have at least one part to `message_name`.

Reply:
Ah, didn't catch that, forgot that `all([]) == True`

Reply:
Updated edbb844.